### ISSUE TITLE
Maps

### ISSUE THREAD

**Issue by christiancosgrove** [BoolClub/ProjectCrusade#44]
With current level sizes, it is easy to get lost. Therefore, I think we should add maps to the game. Unexplored regions won't appear on the map. Specifically, the `Map` will be a GUI element that, e.g., could be displayed on the inventory screen. Is this a good feature?

I'll work on this feature on a separate branch.

**Comment by Authman2** [comment]
Yeah I agree. We also need to decide on the number of floors that the player has to go through.

**Comment by Xykron** [comment]
Is the map only going to be available in the inventory, or will there be some sort of minimap functionality?

**Comment by Authman2** [comment]
I was thinking that it would be available on the inventory or something, because a mini map might get in the way of the game.

**Comment by christiancosgrove** [comment]
I put it on the inventory screen also because it's costly to render. At least at the moment, rendering the map involves a single draw call for every pixel in the image. Better do this when FPS doesn't really matter.
